src/BinDeps.jl

```diff
@@ -561,6 +561,21 @@ function eval_anon_module(context, file)
     return
 end
 
+"""
+    glibc_version()
+
+For Linux-based systems, return the version of glibc in use. For non-glibc Linux and
+other platforms, returns `nothing`.
+"""
+function glibc_version()
+    Compat.Sys.islinux() || return
+    libc = ccall(:jl_dlopen, Ptr{Void}, (Ptr{Void}, UInt32), C_NULL, 0)
+    ptr = Libdl.dlsym_e(libc, :gnu_get_libc_version)
+    ptr == C_NULL && return # non-glibc
```

Comment:
Personally, I don't like the `... || return` and `... && return` style, but that's a total style thing and not something I'm asking you to change.  Just whining.  :)